homeassistant/components/vacuum/__init__.py

```diff
@@ -88,7 +90,8 @@
 SUPPORT_STATUS = 128
 SUPPORT_SEND_COMMAND = 256
 SUPPORT_LOCATE = 512
-SUPPORT_MAP = 1024
+SUPPORT_CLEAN_SPOT = 1024
```

Comment:
because no platform is using the `SUPPORT_MAP` yet... I put it there in order to see if anyone takes the iniciative and implements it ;-)